mico-core/src/main/java/io/github/ust/mico/core/dto/PrometheusResponse.java

```diff
@@ -53,6 +78,7 @@ public boolean wasSuccessful(){
     @JsonProperty("data")
     @SuppressWarnings("unchecked")
     private void unpackNested(Map<String, Object> data) {
+        //List<ResultData> resultListTest = data.get("result");
```

Comment:
Remove comment or improve this code like the ToDo suggests.

Reply:
Refactoring of this class is part of #427 and all changes were made for testing purposes only (but should not be in this PR). Changes in this class are reverted.